### ISSUE TITLE
[Request] Category count(s)

### ISSUE THREAD

**Issue by rin506** [Hanhula/projectbrigid#48]
In chatting with Elizabread about category structures, I realized having a Number of Categories in Brigid could be useful, either as a neat stat or as a way to help organize. Other useful category counts:
- # of articles per category
- # of layers of category nesting
- categories with neither sub-categories nor articles in them
- how many articles/sub-categories are in each category

**Comment by Hanhula** [comment]
We currently aren't retrieving categories; this will come in a little bit later with the category manager. I'll set this to todo for then!

**Comment by Hanhula** [comment]
@Tulonsae Absolutely, but I can't say yet what format those will be displayed in! I'm hoping to allow easy sorting/arranging of categories at this time.
